### PR TITLE
#159 improve notification component

### PR DESCRIPTION
**Pull request by Mikivitkovski**

### CODE REVIEW

**Review by edichoska — APPROVED** [review]
@Mikivitkovski  this looks great! Lets wait for the captcha PR to be approved first.

File: components/reusable-components/turnstile-captcha/CaptchaWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  theme?: 'auto' | 'light' | 'dark';
+  language?: string;
+  tabIndex?: number;
+}
+
+const CaptchaWidget: React.FC<CaptchaProps> = ({
+  onSuccess,
+  onError,
+  onExpired,
+  siteKey = '0x4AAAAAAAWq36_j09RgOKQR',
```
Comment (stoimenov95) [inline components/reusable-components/turnstile-captcha/CaptchaWidget.tsx:18]: We can keep this in some `.env` file.

File: components/reusable-components/turnstile-captcha/CaptchaWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        widgetIDRef.current = id;
+      } catch (error) {
+        isErrorRef.current = true;
+        if (onError) onError();
+      }
+    };
+
+    const loadCaptchaScript = () => {
+      const script = document.createElement('script');
+      script.src = 'https://challenges.cloudflare.com/turnstile/v0/api.js?render=explicit';
```
Comment (stoimenov95) [inline components/reusable-components/turnstile-captcha/CaptchaWidget.tsx:52]: This value can be kept in `.env` as well.

**Review by stoimenov95 — APPROVED** [review]
Regarding the notification - everything looks great Miki!
However, I am not sure why is the Captcha code in this PR, but you can acknowledge the comments I left here and resolve them in the Captcha ticket.

**Comment by edichoska** [comment]
@Mikivitkovski merge this when you have he chance 🍀

**Review by dichoska — CHANGES_REQUESTED** [review]
we are closing this PR since Miki already created another one which contained only the changes in the notification component

**Pull request closed without merging**